src/monggregate/operators/accumulators/avg.py

```diff
@@ -75,12 +75,32 @@
 
 class Average(Accumulator):
     """
-    Creates a $avg expression.
+    Abstration of MongoDB $avg operator that returns the average value of numeric values.
 
     Attributes
-    ------------------------
-        - expression : Any valid expression
-
+    ----------
+    expression : Any valid expression
+
+    Online MongoDB documentation:
+    -----------------------------
+    Returns the average value of the numeric values.
+    $avg ignores non-numeric values.
+
+        * $avg is available in these stages:
+        * $addFields (Available starting in MongoDB 3.4)
```

Comment:
Problème de formatting ici. Soit "$avg is available..." devrait être plus à gauche, soit les lignes en dessous devraient être indentées.